docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -212,14 +246,36 @@ properties must be defined.
 The type of kNN algorithm to use. Currently only `hnsw` is supported.
 
 `m`:::
-(Required, integer)
+(Optional, integer)
```

Comment:
The previous paragraph has a sentence that needs to be updated to reflect the new optionality of these parameters. The sentence is:

`When index_options is provided, all of its properties must be defined.`